api/src/main/java/com/aldoborrero/tinder/api/model/TinderLog.java

```diff
@@ -0,0 +1,16 @@
+package com.aldoborrero.tinder.api.model;
+
+import retrofit.RestAdapter;
+
+public enum  TinderLog {
+    NONE;
+
+    public static RestAdapter.Log parseToRetrofit (TinderLog log) {
```

Comment:
In here we're relaying in Retrofit's log system. But what happens if we want to use our custom one? I prefer to create constants and a Logger object that possibly encapsulates other loggers (as Retrofit one).